PostHog/PostHogQueue.swift

```diff
@@ -45,7 +45,7 @@ class PostHogQueue {
         self.storage = storage
         self.api = api
         self.reachability = reachability
-        fileQueue = PostHogFileBackedQueue(url: storage.url(forKey: .queue))
+        fileQueue = PostHogFileBackedQueue(queue: storage.url(forKey: .queue), oldQueue: storage.url(forKey: .oldQeueue))
```

Comment:
Personally I would choose to have the "legacy" logic as it's own standalone class, which takes the fileQueue as an argument. That way we can keep it clean and separate. Not going to block you so up to you if you agree.

Would just make it easier in the future to know what the code is for, and simpler to read.